website/docs/configure_hydra/workdir.md

```diff
@@ -55,3 +55,15 @@ hydra:
         item_sep: ','
         exclude_keys: []
 ```
+
+### Customizing outputs with substitution through the CLI 
+
+Outputs can also be configured through the CLI, like any other configuration.
+
+> python train.py model.nb_layers=3 hydra.run.dir=3_layers
+
+This feature can become really powerful to write multiruns without boilerplate using substitution.
+
+> python train.py --multirun model.nb_layers=1,2,3,5 hydra.sweep.dir=multiruns/layers_effect hydra.sweep.subdir=\${model.nb_layers}
```

Comment:
This would have been better as a code block:

    ```  
    python train.py --multirun model.nb_layers=1,2,3,5 hydra.sweep.dir=multiruns/layers_effect hydra.sweep.subdir=${model.nb_layers}
    ```

Reply:
following our discussion:
the objective here is to render `\${model.nb_layers}`, with the prefix backslash.